age/src/cli_common.rs

```diff
@@ -31,17 +31,18 @@ pub fn get_config_dir() -> Option<PathBuf> {
 
 /// Reads identities from the provided files if given, or the default system
 /// locations if no files are given.
-pub fn read_identities<E, F, G, H>(
+#[rustfmt::skip]
```

Comment:
This is annoying, but without it, `rustfmt` removes the inline `#[cfg(feature = "ssh")]`.